excel/Ballerina.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+org= "ballerinax"
+name= "microsoft.excel"
+version= "0.11.0"
```

Comment:
In master branch, version should be in SNAPSHOT

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-microsoft.excel/commit/7221c821c3249e73220eb70ef191a826e7684036